app/src/main/java/com/example/poste/utils/ValidationUtils.java

```diff
@@ -17,16 +19,24 @@ public class ValidationUtils {
      */
     public static boolean validateNames(Context context, String firstName, String lastName) {
         if (firstName.isEmpty()) {
-            Toast.makeText(context, "Please enter your first name!", Toast.LENGTH_SHORT).show();
+            Toast.makeText(context,
+                    R.string.enter_first_name,
+                    Toast.LENGTH_SHORT).show();
             return false;
         } else if (lastName.isEmpty()) {
-            Toast.makeText(context, "Please enter your last name!", Toast.LENGTH_SHORT).show();
+            Toast.makeText(context,
+                    R.string.enter_last_name,
+                    Toast.LENGTH_SHORT).show();
             return false;
         } else if (firstName.length() > 32) {
-            Toast.makeText(context, "First name must be at most 32 characters!", Toast.LENGTH_SHORT).show();
+            Toast.makeText(context,
+                    R.string.first_name_length,
```

Comment:
Suggest name like "error_first_name_length" so it is clear this is an error message of some type (as opposed to just a length)